src/makeRobotAccountant.js

```diff
@@ -5,8 +5,18 @@
  * @return {function}
  */
 
-function makeRobotAccountant() {
-  // write code here
+function makeRobotAccountant(a) {
+  let count = 0;
+
+  return firstNumber => secondNumber => {
```

Comment:
The function `getSum` is not defined. Instead of using a single return statement with two arrow functions, create a new function `getSum` that takes two arguments and returns their sum, then return `getSum` from `makeRobotAccountant`.